azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/CallCompositeActivity.kt

```diff
@@ -217,9 +219,14 @@ internal class CallCompositeActivity : AppCompatActivity() {
     }
 
     override fun onBackPressed() {
-        val fragment = supportFragmentManager.fragments.first()
-        if (fragment !== null) {
-            (fragment as BackNavigation).onBackPressed()
+
+        if (!supportFragmentManager.fragments.isEmpty()) {
```

Comment:
Should we just change next line to:
val fragment = supportFragmentManager.fragments.firstOrNull()

Reply:
Addressed. Thanks :)